testrpc/test_metadata.go

```diff
@@ -0,0 +1,5 @@
+package testrpc
+
+func GetTestMetaData() string {
```

Comment:
does that mean we will have to seed the "Test data" with hex data each time?
or is it somehow possible to set this more "human friendly"?

Reply:
this metadata is for the reply of the metadata rpc call which has to be hex. But we can also encoded it on the fly from human friendly form. To keep it simple for now, I'm just using a response I got from our substrate chain. But we can change this in the future for sure.